src/Makefile

```diff
@@ -127,7 +127,7 @@ else
 ifeq ($(OS), Darwin)
 CG_LLVMLINK += $(LLVM_LDFLAGS) -lLLVM
 else
-CG_LLVMLINK += $(LLVM_LDFLAGS) -lLLVM-13jl
+CG_LLVMLINK += $(LLVM_LDFLAGS) $(shell $(LLVM_CONFIG_HOST) --libs  --link-shared)
```

Comment:
@vtjnash any clue why we didn't do this before?

Reply:
It used to return the wrong answer

Reply:
Or maybe it still does?
```
llvm-config: error: libLLVM-13jl.dll is missing
```

Reply:
Isn't this the issue that on Windows `dll` needs to be next to the library and `llvm-config` lives in `/tools` and not `/bin`?